src/runtime/HalideRuntime.h

```diff
@@ -723,30 +723,28 @@ typedef enum halide_target_feature_t {
     halide_target_feature_sse41 = 4,  ///< Use SSE 4.1 and earlier instructions. Only relevant on x86.
     halide_target_feature_avx = 5,  ///< Use AVX 1 instructions. Only relevant on x86.
     halide_target_feature_avx2 = 6,  ///< Use AVX 2 instructions. Only relevant on x86.
-    halide_target_feature_fma = 7,  ///< Enable x86 FMA instruction
-    halide_target_feature_fma4 = 8,  ///< Enable x86 (AMD) FMA4 instruction set
-    halide_target_feature_f16c = 9,  ///< Enable x86 16-bit float support
+    halide_target_feature_avx512 = 7,  ///< Use AVX 512 instructions (the subset supported by Cannonlake). Only relevant on x86.
```

Comment:
It seemed like a good idea to put avx512 next to avx2. I could also move it to the end.